certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -35,20 +35,9 @@ class MultipleVhostsTest(util.ApacheTest):
     def setUp(self):  # pylint: disable=arguments-differ
         super(MultipleVhostsTest, self).setUp()
 
-        from certbot_apache.constants import os_constant
-        orig_os_constant = os_constant
-        def mock_os_constant(key, vhost_path=self.vhost_path):
-            """Mock default vhost path"""
-            if key == "vhost_root":
-                return vhost_path
-            else:
-                return orig_os_constant(key)
-
-        with mock.patch("certbot_apache.constants.os_constant") as mock_c:
-            mock_c.side_effect = mock_os_constant
-            self.config = util.get_apache_configurator(
-                self.config_path, None, self.config_dir, self.work_dir)
-            self.config = self.mock_deploy_cert(self.config)
+        self.config = util.get_apache_configurator(
```

Comment:
I looked quickly into it previously, and it seemed like a lot of work, as a large chunk of the "old tests" make assumptions based on the distro. Now when taking a closer look I was able to separate the Debian tests with minimal duplication, so this is now taken care of.

Reply:
Awesome. Thanks for doing this.

I can imagine it might have been a little tedious but I think it helps clean things up and make the purpose of different files more clear.